auslib/admin/views/releases.py

```diff
@@ -348,40 +352,14 @@ def _put(self, release, changed_by, transaction):
         return Response(status=201, response=json.dumps(dict(new_data_version=data_version)))
 
 
-class ReleaseHistoryView(HistoryAdminView):
+class ReleaseHistoryView(HistoryView):
     """/releases/:release/revisions"""
 
-    def get(self, release):
-        releases = dbo.releases.getReleases(name=release, limit=1)
-        if not releases:
-            return Response(status=404,
-                            response='Requested release does not exist')
-        release = releases[0]
-        table = dbo.releases.history
+    def __init__(self):
+        super(ReleaseHistoryView, self).__init__(dbo.releases)
 
-        try:
-            page = int(request.args.get('page', 1))
-            limit = int(request.args.get('limit', 10))
-            assert page >= 1
-        except (ValueError, AssertionError) as e:
-            self.log.warning("Bad input: %s", json.dumps(e.args))
-            return Response(status=400, response=json.dumps({"data": e.args}))
-        offset = limit * (page - 1)
-        total_count = table.t.count()\
-            .where(table.name == release['name'])\
-            .where(table.data_version != null())\
-            .execute()\
-            .fetchone()[0]
-
-        revisions = table.select(
-            where=[
-                table.name == release['name'],
-                table.data_version != null()
-            ],
-            limit=limit,
-            offset=offset,
-            order_by=[table.timestamp.desc()],
-        )
+    def __process_revisions(self, revisions):
```

Comment:
Nit: don't use __ - that's usually reserved for language-level stuff. A single _ will suffice, and is sometimes used to try to signal that things are private methods.